### DISCUSSION TITLE
Built-in backends.

### DISCUSSION THREAD

**Discussion by alex-oleshkevich** [encode/broadcaster#118] [General]
@encode/maintainers do we want to ship with many backends by default? I am in doubt. Users can now configure their own backends and use with broadcaster, so I think that we can limit built-in backends to ones we already have and not accept any new ones.

**Comment by lovelydinosaur** [comment]
> I think that we can limit built-in backends to ones we already have and not accept any new ones.

Makes sense yep. We could also link to gists or repositories with third party options if needed.
